pkgs/by-name/pa/packet/package.nix

```
@@ -0,0 +1,75 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cargo,
+  meson,
+  ninja,
+  pkg-config,
+  protobuf,
+  rustPlatform,
+  rustc,
+  wrapGAppsHook4,
+  cairo,
+  dbus,
+  gdk-pixbuf,
+  glib,
+  gtk4,
+  libadwaita,
+  pango,
+  blueprint-compiler,
+  desktop-file-utils,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "packet";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "nozwock";
+    repo = "packet";
+    rev = finalAttrs.version;
```

Tagged releases should use the tag field
```suggestion
    tag = finalAttrs.version;
```